build.rs

```diff
@@ -0,0 +1,11 @@
+use std::process::Command;
+
+// remove husky configuration from .git/config if it exists
+fn main() {
+    Command::new("git")
+        .arg("config")
+        .arg("--unset")
+        .arg("core.hooksPath")
+        .status()
+        .expect("core.hooksPath failed to reset. You should manually run git config --unset core.hooksPath");
+}
```

Comment:
So with this update when a dev runs `cargo build` (per the directions in CONTRIBUTING.md) the following steps occur in this order:
1. `build.rs` runs & removes `hooksPath = .husky" from the dev's git config (per the above)
2. rusty-hook is installed as a dependency per `Cargo.toml`
3. rusty-hook's post-build script installs the git commit hooks in the hidden `.git/hooks` dir

So, technically, I didn't reduce the number of commit hooks that are installed, but they are hidden in the `.git/hooks` folder as they should be.

I *could* remove the extra git hooks, but I would need to add more commands to `.build.rs` & it feels like overkill.

Reply:
oh nice, this is great. i like this solution